student-test/assignment1/Test_validate_username.c

```diff
@@ -15,8 +15,8 @@
 void test_validate_my_username()
 {
     /**
-     * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
+     * Below code verifies /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    TEST_ASSERT_EQUAL_STRING_MESSAGE(my_username(),malloc_username_from_conf_file(),"Validating usernames");
```

Comment:
@Induja21 what happens to the string allocated by `malloc_username_from_conf_file()`?

Reply:
Hi @dwalkes , since  memory is allocated dynamically it should have been freed here , which was missed in the assignment :/ . Thanks for pointing it out